pkgs/by-name/de/deutex/package.nix

```
@@ -0,0 +1,41 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  autoreconfHook,
+  pkg-config,
+  libpng,
+}:
+
+stdenv.mkDerivation rec {
+  pname = "deutex";
+  version = "5.2.3";
+
+  src = fetchFromGitHub {
+    owner = "Doom-Utils";
+    repo = "${pname}";
+    rev = "v${version}";
```

```suggestion
    repo = "deutex";
    tag = "v${version}";
```
use tag; use of pname like this is an antipattern and breaks overrides